openc3-init/plugins/packages/openc3-tool-cmdsender/src/tools/CommandSender/CommandParameterEditor.vue

```diff
@@ -160,10 +180,17 @@ export default {
 }
 </script>
 <style scoped>
+/* This allows Value or State selection to be wider and show state names */
+.container :deep(.v-select__selections) {
+  width: auto;
+}
 .v-overflow-btn {
   margin-top: 0px;
 }
 .container {
   padding: 0px;
 }
+.hazardous :deep(.v-select__selection) {
+  color: rgb(255, 220, 0);
+}
```

Comment:
I only color the state name ... didn't try to figure out how to push the astro badge into the state